.github/workflows/ci.yaml

```diff
@@ -102,7 +102,7 @@ jobs:
     steps:
       - uses: actions/checkout@v4
         with:
-          repository: "dcherian/xarray"
+          repository: "pydata/xarray"
```

Comment:
is there a reason why you clone `xarray` `main`, then install `flox` from a github link?

I'd have expected to clone `flox`, then install `xarray` from github (or from the nightlies), and then finally install `flox` from local.

Reply:
This job runs the *xarray* tests, that's why I clone it.